### DISCUSSION TITLE
Ordered wrong header - help confirming pins

### DISCUSSION THREAD

**Discussion by IronicMeme** [JanM321/esphome-lg-controller#61] [Q&A] [answered]
Hi,
I ordered 10x Tiny from JLCPCB and seeing as I was half asleep when I ordered them, I replaced the female headers with male headers that were out of stock at the time of ordering. I have a bunch of jumper cables so I should be able to simply connect the relevant pins to the ESP.

I've connected the 3P pin on the Tiny to the 3v3 pin on my ESP, and the pin underneath the 3P to the ground pin, however I am unable to get the ESP to power up when connected to the LG remote connector. 

- Do I have the correct pin for ground?
- Is it normal for the A1027/2 chip to get quite hot when connected to the air conditioner header? (I was able to receive data from the AC but couldn't send anything - I was powering the ESP via USB as I couldn't get it powered from the LG)

![IMG_6670](https://github.com/JanM321/esphome-lg-controller/assets/35889739/d40e6f92-3465-4cd8-81d2-bad85cad8121)
![IMG_6671](https://github.com/JanM321/esphome-lg-controller/assets/35889739/4deb402e-1cc3-43f1-8d40-72c50e09eff6)
![IMG_6669](https://github.com/JanM321/esphome-lg-controller/assets/35889739/ec6a46f2-af82-4c68-9165-f7a81ad78c6e)

**Comment by JanM321** [comment, marked as answer]
The +12V / LIN / GND labels on the PCB refer to the 3-pin connector to the AC, not the ESP pins.

See the picture here: https://github.com/JanM321/esphome-lg-controller/blob/main/hardware/schematic.png

The top pin (above TXD next to the antenna area) is 3v3. The first one on the other side of the board is a GND.

**Reply by IronicMeme** [reply]
Thanks! I tried to look into the Gerber files to find out which pin was which but couldn't understand it haha
I was able to get it to power the ESP using the correct pins and receive data but now I am getting invalid checksum errors. I'll post more details in an issue.

**Comment by JanM321** [comment]
Also, the pins on that ESP board don't match the schematic and what I see here: https://github.com/JanM321/esphome-lg-controller/blob/main/hardware-tiny/pcb.jpg If you zoom in on that picture you can see the 3V3 in the top left corner.
